amplipi/updater/static/index.html

```diff
@@ -105,6 +110,40 @@ <h1><span class="text-white">Ampli</span><span class="text-danger">Pi</span></h1
                 </div>
               </div>
             </div>
+
+            <div class="tab-pane fade text-left" id="admin-settings" role="tabpanel" aria-labelledby="admin-settings-tab">
+              <script>
+                async function getPersist() {
+                  const response = await fetch("/settings/persist_logs", { method: "GET" });
+                  const data = await response.json();
+                  let checkbox = document.getElementById("persist_logs_checkbox");
+                  checkbox.checked = data;
+                }
+
+                async function setPersist() {
+                  const checkbox = document.getElementById("persist_logs_checkbox");
+                  const response = await fetch("/settings/persist_logs", {
+                    method: "POST",
+                    headers: { 'Content-Type': 'application/json' },
+                    body: JSON.stringify({ "value": !checkbox.checked }) // Explicitly set to the opposite of the present setting to ensure that two users hitting the checkbox don't toggle it twice
+                  });
+                }
+
+                window.onload = getPersist;
+              </script>
```

Comment:
potentially belongs in the js folder, currently it lives here because it was easiest to develop stuff inline